real-main/app/handlers/appsync/handlers.py

```
@@ -321,6 +331,20 @@ def set_user_details(caller_user, arguments, **kwargs):
     if privacy_status is not None:
         caller_user.set_privacy_status(privacy_status)
 
+    if current_location is not None:
+        latitude = Decimal(str(current_location.get('latitude')))
+        longitude = Decimal(str(current_location.get('longitude')))
+        accuracy = Decimal(str(current_location.get('accuracy', 0)))
+
+        current_location = {"latitude": latitude, "longitude": longitude, "accuracy": accuracy}
```

A few thoughts on this:

- Since using `Decimal` is essentially a dynamodb requirement, I think we should probably put this conversion 'closer' to dynamo. Probably in the method `set_user_details()` of `real-main/app/models/user/dynamo/base.py`
- if `accuracy` isn't set by the client, let's leave it not set in dynamo (don't fill in a zero)
- in the interest of 'fail early and often' I would do `current_location['latitude']`, not `current_location.get('latitude')`
- to convert a float to a decimal, I think this is a better formulation:

```py
from decimal import BasicContext
my_float = 6/7
my_decimal = BasicContext.create_decimal(my_float).normalize()
```

I'm going to go through other places in this project where we use `Decimal` and change them to initialize decimals in this way (will do that in a separate commit, not part of this PR)